### PR TITLE
feat: clarified language in localization tutorial using feedback

### PR DESCRIPTION
**Pull request by nicolas-ethyca**

### CODE REVIEW

File: pages/tutorials/consent-management-configuration/privacy-notices-localization.mdx [hunk 1]
```diff
@@ -55,4 +55,7 @@ Defines the locations in which this privacy notice will be displayed. You would
 
 <Screenshot img="/assets/img/tutorials/consent-management-config-location-France.png" description="Privacy Notice Configure Locations" mw="500" />
 
-Next, we'll configure privacy experiences for our available notices.
\ No newline at end of file
+#### Fides Cookie Keys
+To reuse your existing tag manager or Javascript integrations, you should reuse the same key for all versions of notice. 
+
+For example, you can create multiple notices that use the same `data_sales_and_sharing` key for integrations, but different locations for different languages.
```
Comment (NevilleS) [inline pages/tutorials/consent-management-configuration/privacy-notices-localization.mdx:61]: Can you update the `Privacy Notice Configuration` section above instead? In there you say "...a unique key for cookie consent configuration" but now that conflicts with this 😬 

I'd recommend rewriting that section to basically just suggest that they copy the data uses and key from an existing notice, and then just change the location.

**Review by NevilleS — CHANGES_REQUESTED** [review]
Bit confusing!

File: pages/tutorials/consent-management-configuration/privacy-notices-localization.mdx (hunk 1 above)
Comment (nicolas-ethyca) [reply pages/tutorials/consent-management-configuration/privacy-notices-localization.mdx]: @NevilleS how about now?

**Review by NevilleS — APPROVED** [review]
This change will help, thanks, so I'd vote to merge it in.

That saidI think it's still confusing though. We don't _really_ explain how to duplicate an existing notice and configure different, location-specific language, so if we tested this on a person I think they'd still be confused. I'm not exactly sure *what* I'm looking for, but it'd probably be something that covers:
* why you'd want to create a location-specific notice
* how you'd duplicate an existing notice and configure up
* why you'd reuse the same notice key (so you don't need to update GTM/etc.)
* etc.

...but perfection is the enemy of good so let's make this change to make it clear that this is optional thanks

**Pull request merged**